go.mod

```diff
@@ -22,4 +22,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20210914134654-45c375935ced
```

Comment:
Friendly reminder

Reply:
thanks ;) I removed it in https://github.com/codeready-toolchain/member-operator/pull/297/commits/a0195f7a0c7360a72c0f02eeac5328df134298e4